test/utils/prune_namespaces.go

```diff
@@ -0,0 +1,14 @@
+package main
```

Comment:
could we call the folder like `scripts` or similar ?
I find `utils` as a test package name ...

Reply:
Sure. Renamed to scritps.